ui/index.html

```diff
@@ -0,0 +1,113 @@
+  <header class="header">
+    <nav>
+      <div class="nav-container">
+        <div class="btn">
+          <div class="bar"></div>
+          <div class="bar"></div>
+          <div class="bar"></div>
+        </div>
+      </div>
+      <ul>
+        <div class="logo">Maintenance Tracker</div>
+        <li>
+          <a href="#">home</a>
+        </li>
+        <li>
+          <a href="#about-section">about</a>
+        </li>
+        <li>
+          <a href="signin.html">sign in</a>
+        </li>
+      </ul>
+    </nav>
+    <div class="banner">
+      <div class="banner-center">
+        <h1>Maintenance Tracker</h1>
+        <h3>Repairing one day at a time</h3>
+        <div>
+          <a class="button" href="signin.html">Sign In</a>
+          <a class="button" href="signup.html">Sign Up</a>
+        </div>
+        <div class="banner-icons">
+          <a href="">
+            <i class="fab fa-facebook"></i>
+          </a>
+          <a href="">
+            <i class="fab fa-instagram"></i>
+          </a>
+        </div>
+      </div>
+    </div>
+  </header>
+  <!--header-section-end-->
+
+  <!-- About Section -->
+  <section class="about" id="about-section">
+    <div class="about-center">
+      <div class="about-heading">
+        <h2>About</h2>
+      </div>
+      <article class="about-article">
+        <div class="about-icon">
+            <i class="fas fa-wrench"></i>
+        </div>
+        <div class="about-text">
+          <h3>Quick Maintenance</h3>
+          <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna
+            aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat
+          </p>
+        </div>
+      </article>
+      <article class="about-article">
+        <div class="about-icon">
+            <i class="fas fa-wrench"></i>
+        </div>
+        <div class="about-text">
+          <h3>Durable Repairs</h3>
+          <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna
+            aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat
+          </p>
+        </div>
+      </article>
+      <article class="about-article">
+        <div class="about-icon">
+            <i class="fas fa-screwdriver"></i>
+        </div>
+        <div class="about-text">
+          <h3>Effective</h3>
+          <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna
+            aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat
+          </p>
+        </div>
+      </article>
+    </div>
+  </section>
+  <!-- About Section End -->
+
+  <!--Footer Section -->
+  <footer>
+    <p>copyright&copy;maintenance tracker 2018. All rights reserved.</p>
```

Comment:
There should be spaces between `copyright`, `&copy;` and `maintenance`

Reply:
Alright, I will im plement that